src/datacenters/v1/README.md

```diff
@@ -0,0 +1,216 @@
+# v1 datacenter schema
+
+The v1 datacenter schema is designed to allow operators to control resources in three
+different scopes:
+
+- Datacenter-scoped resources
+- Environment-scoped resources
+- Application-scoped resources
+
+By combining these three scopes, operators can create robust datacenter templates that
+can be used to host and integrate virtually any application in any style of cloud
+environment.
+
+## Variables
+
+Though most of each datacenter schema is self-contained, there are some cases where datacenters
+templates need additional input from users before they can be used to power a datacenter. These
+inputs are called variables:
+
+```yml
+variables:
+  account:
+    # Variables can be arcctl resources
```

Comment:
Resources should probably be a link and it could be cool to also show off another resource type that is more well understood like a dnsZone.